step-plans/step-plans-core/src/test/java/step/core/repositories/AbstractRepositoryTest.java

```diff
@@ -42,7 +43,7 @@ public ArtefactInfo getArtefactInfo(Map<String, String> repositoryParameters) {
             }
 
             @Override
-            public TestSetStatusOverview getTestSetStatusOverview(Map<String, String> repositoryParameters) {
+            public TestSetStatusOverview getTestSetStatusOverview(Map<String, String> repositoryParameters, ObjectPredicate objectPredicate) {
```

Comment:
@david-stephan Sorry for the misunderstanding. My proposition was to pass the AbstractContext in both methods instead of ObjectPredicate. It is not required currently but could be useful in the future.

Reply:
@jeromecomte Did some more check but to use the object predicate we would have to pass the GlobalContext (as AbstractContext) but also the Session which is required to apply the predicate. For now we'll merge as it is.